plugins/GSdx/GSdx.rc

```diff
@@ -277,7 +277,7 @@ BEGIN
     GROUPBOX        "Hardware Renderer Settings",IDC_STATIC,6,121,230,140,BS_CENTER
     CONTROL         "Accurate DATE",IDC_ACCURATE_DATE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,11,132,63,10
     CONTROL         "Allow 8-Bit Textures",IDC_PALTEX,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,77,132,76,10
-    CONTROL         "Large Framebuffer",IDC_LARGE_FB,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,158,132,75,10
+    CONTROL         "CBA",IDC_LARGE_FB,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,158,132,75,10
```

Comment:
Not a fan of the short name.

Reply:
I have changed it to Cons.Buffer unless you prefer something else?

Reply:
I prefer "Conservative Buffer Allocation" which is why I told you to change it to that :P 
Move the settings around if you have to.

I'd also prefer not to invert it by setting read but by setting write so as to retain the original setting semantics in the config.
In other words, when you got to save it in the gui, read the opposite value of the checkbox. If it's set write 0 and if it's unset write 1.

Reply:
Oh yeah this is a sin, not sure what i was thinking. I'll fix that write.